### PR TITLE
Php 8 2 preparation

### PR DESCRIPTION
**Pull request by ChipsVII**

### CODE REVIEW

File: src/Classes/StringUtil.php [hunk 1]
```diff
@@ -127,4 +168,57 @@ public static function formatFileSize(int $bytes): string
...
+        $characters = 'abcdefghijklmnopqrstuwxyzABCDEFGHIJKLMNOPQRSTUWXYZ0123456789_-';
+        $randstring = '';
+        for ($i = 0; $i < $length; ++$i) {
+            $randstring .= $characters[random_int(0, \strlen($characters) - 1)];
+        }
+
+        return $randstring;
+    }
+
+    public static function getFormStorageDataValueAsString($mixed): string
```
Comment (ChipsVII) [inline src/Classes/StringUtil.php]: @NodokaMurmevent Why is this here ? Shouldn't it belong to the https://github.com/Web-Ex-Machina/contao-form-data-manager bundle ?

File: src/Classes/Files.php [hunk 2]
```diff
@@ -100,7 +98,7 @@ public static function processDzFileUpload($file, $folder)
             $session->set(sprintf('dzupload_%s_%s_%s', $_POST['dzuuid'], $_POST['dzchunkindex'], $_POST['dztotalchunkcount'] - 1), true);
 
             $blnMerge = true;
-            for ($i = 0; $i < $_POST['dztotalchunkcount']; $i++) {
+            for ($i = 0; $i < $_POST['dztotalchunkcount']; ++$i) {
```
Comment (ChipsVII) [inline src/Classes/Files.php:101]: @NodokaMurmevent will it work the same as `$i++` ? 
Will `$i` still be `0` on the very first pass ? Or already be `1` ?

Comment (NodokaMurmevent) [reply src/Classes/Files.php:101]: in a while of a for it’s change nothing, it’s start at zero. I changed this (with rector) because ++$i is pre-increment and it's average 10% faster than $i++ post-increment : 

```PHP
for($i = 0;$i<3;++$i){var_dump($i);}
/*
int(0)
int(1)
int(2)
*/
for($i = 0;$i<3;$i++){var_dump($i);}
/*
int(0)
int(1)
int(2)
*/

```

File: src/Classes/StringUtil.php (hunk 1 above)
Comment (NodokaMurmevent) [reply src/Classes/StringUtil.php]: it’s used here [contao-form-data-manager UiListener.php line 212](https://github.com/Web-Ex-Machina/contao-form-data-manager/blob/e5b953f772bd990656a58c109dd117c863ce1b04/src/EventListener/PersonalDataManager/UiListener.php#L212)

but it's true, if the form data manager is the alone to use it, we can move it in this bundle.

File: src/Classes/Files.php (hunk 2 above)
Comment (ChipsVII) [reply src/Classes/Files.php:101]: Okaaay !

File: src/Classes/StringUtil.php (hunk 1 above)
Comment (ChipsVII) [reply src/Classes/StringUtil.php]: I reckon no one will call this function unless the FDM bundle is installed, in which case they could call it from there.
I'd prefer this function to be moved to the FDM bundle
